tests/integration/orcid/test_orcid_push.py

```diff
@@ -152,3 +153,73 @@ def _push_record_with_orcid(_recid, _orcid, tk, _put_code=None):
     monkeypatch.setattr(tasks, 'push_record_with_orcid', _push_record_with_orcid)
 
     attempt_push(orcid, recid, 'fake-token')
+
+
+def test_feature_flag_orcid_push_whitelist_regex_none():
```

Comment:
If I'm not mistaken this test test `re`, not really the code you wrote?

Reply:
True. The idea is not to test the code but the possible values for the whitelist that I suggest in the config file: `^$`, `.*`, `^(val1|val2)$`